src/status_im2/contexts/profile/profiles/views_v2.cljs

```diff
@@ -0,0 +1,123 @@
+(ns status-im2.contexts.profile.profiles.views-v2
+  (:require [clojure.string :as string]
+            [quo.core :as quo]
```

Comment:
please don't use any components or code that lives in quo. namespaces. 
We strictly want to use quo2 code 👍

Reply:
I think that might also be why you needed to adjust so many icon files as you are working from mostly old code :/

Reply:
Changed to quo2 most of things I can. Please review again.

Reply:
Hi @nguyentruongky, the changes you made look great. I would prefer to discuss with you before reviewing again as there are some ways that I should be able to help you find the correct code/components in the codebase. let's chat on discord 👍

Reply:
Thanks @J-Son89 How can I reach out to you on discord. Just registered with username `adventurous_beagle_60958`